README.md

```diff
@@ -74,19 +74,19 @@ pull requests.
 
 You can simply run from the git checkout now:
 ```
-$ sudo ./sosreport -a
+$ sudo ./bin/sosreport -a
 ```
 
 Or, if you only have python3 installed:
 ```
-$ sudo python3 ./sosreport -a
+$ sudo python3 ./bin/sosreport -a
```

Comment:
We've moved to python3-only. Let's converge this and the previous section about running from the checkout.